wpilibc/sim/src/PWM.cpp

```diff
@@ -203,6 +203,10 @@ void PWM::SetPeriodMultiplier(PeriodMultiplier mult) {
   // Do nothing in simulation.
 }
 
+void PWM::SetZeroLatch() {
+  // Do nothing in simulation.
```

Comment:
The athena version of PWM calls a HAL function. I can't tell what it does. Since it originally wasn't used by the PWM motor controllers, I added a no-op stub here so the sim motor controller classes are closer to what's in athena with no change in behavior.